Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ListWSController.java

```diff
@@ -335,6 +335,7 @@ public String getWorkspaceDetails(@PathVariable("workspaceid") String workspacei
 		
 		//Including a condition to check if the workspace is not deactive. If the workspace is deactive adding attribute to make delete button disabled
 		model.addAttribute("isDeactivated", wsManager.getDeactiveStatus(workspaceid));
+        model.addAttribute("isArchived", wsManager.getArchiveState(workspaceid));

```

Comment:
actually, we use 4 spaces as well (default in eclipse is 6 spaces though), but we haven't been adopting strict formatting guidelines until now. For new classes you write please use 4 spaces. Thanks.